doc/man3/BIO_read.pod

```diff
@@ -54,11 +55,13 @@ BIO_write() attempts to write I<len> bytes from I<buf> to BIO I<b>.
 BIO_puts() attempts to write a NUL-terminated string I<buf> to BIO I<b>.
 
 =head1 RETURN VALUES
-
 BIO_read_ex() and BIO_write_ex() return 1 if data was successfully read or
 written, and 0 otherwise.
```

Comment:
I would probably split read_ex and write_ex descriptions here. Because for the `BIO_write_ex()` with this change a more precise definition would be: `BIO_write_ex() returns 1 if no error was encountered writing data, 0 otherwise. Write to NULL B<BIO> is not considered as an error.`

Reply:
Done.